packages/core/src/lib/Slider/Slider.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import { ISliderProps } from './types';
+import styles from './Slider.styles';
+
+const { sliderDiv, sliderInput, sliderLabel, sliderP } = styles;
+
+const Slider: React.FC<ISliderProps> = ({
+    rangeId,
+    rangeLabel,
+    rangeMax,
+    rangeMin,
+    rangeValue,
+}) => {
+    const [currentValue, setCurrentValue] = useState(rangeValue);
+
+    const rangeChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        const newRangeValue = Number(event.currentTarget.value);
+        setCurrentValue(newRangeValue);
+    };
+
+    return (
+        <styles.sliderDiv>
+            <styles.sliderLabel htmlFor={rangeId}>
```

Comment:
`sliderDiv` is already de-structured above so you can use it directly instead of `styles.sliderDiv`
Please rename these as mentioned in my above comment 🙌🏻

Reply:
changed name also directly using them now